components/raftstore/src/store/worker/region.rs

```diff
@@ -457,6 +460,9 @@ where
         if overlap_ranges.is_empty() {
             return Ok(());
         }
+        SNAP_COUNTER_VEC
```

Comment:
I don't get it, why add the metrics to `SNAP_COUNTER_VEC`?

Reply:
There are many kind of counter in `SNAP_COUNTER_VEC ` ....

Reply:
In actually, almost all counter metrics in region.rs are include in `SNAP_COUNTER_VEC `.

Reply:
I don't think so. `SNAP_COUNTER_VEC` is used for counting the generated and applied snapshots. overlap is not a valid snapshot type. Neither `delete_by_ingest` is a valid snapshot status.